pulp_ansible/app/models.py

```diff
@@ -178,6 +179,22 @@ class Meta:
         ]
 
 
+class CollectionVersionSignature(Content):
+    """
+    A content type representing a signature that is attached to a content unit
+    # This model might live in pulpcore if it can be abstracted.
+    """
+    TYPE = "collection_signature"
+
+    content = models.ForeignKey(CollectionVersion, on_delete=models.CASCADE, related_name="signatures")
```

Comment:
Is this the signature itself? If so maybe `s/content/signature/`?

Reply:
No this is a reference to the content this signature is for.

Reply:
IIRC on container modeling `content` is the signature base64 (c/c @ipanova) probably best to have specific naming here such as:

```py
signature_content =
collection_version =
```

Reply:
we went with a more explicit naming and the field that is a reference to the content is named `signed_manifest` https://github.com/pulp/pulp_container/blob/signing/pulp_container/app/models.py#L236
the payload of the signature itself is stored in `data` field, we will serialize it to `content` in the live api for the  client compatibility